
<launch>

  <rosparam  command="load"  file="$(find laser_stitcher)/param/basic_settings/basic_default.yaml"/>
  <rosparam  command="load"  file="$(find laser_stitcher)/param/output_settings/output_default.yaml"/>

	<node
      name    = "laser_stitcher"
      pkg     = "laser_stitcher"
      type    = "laser_stitcher"
      args    = ""
      output  = "screen"
    >
  </node>

	<node
      name    = "lidar_ur5_manager"
      pkg     = "laser_stitcher"
      type    = "lidar_ur5_manager"
      args    = ""
    >
  </node>
    
  <node
    name    = "laser_stitcher_client"
    pkg     = "laser_stitcher"
    type    = "laser_stitcher_client"
    args    = ""
    output  = "screen"
  >
  </node>

  <node
      name    = "pointcloud_processing_server"
      pkg     = "pointcloud_processing_server"
      type    = "pointcloud_processing_server"
      args    = "-intensity"
      output  = "screen"
    >
  </node>

</launch>
